tasks/get_sql_logins.ps1

```diff
@@ -0,0 +1,254 @@
+[CmdletBinding()]
```

Comment:
Might be good practice to include full comment based help for the scripts instead of inline comments for the params? That would let folks run `Get-Help` against the script file.

Reply:
The comment based help will work with comments above each parameter. I added a full comment help block at the end of the script to ensure things like summary and description work as well.